sense-app/src/main/kotlin/com/exactpro/th2/sense/app/Microservice.kt

```diff
@@ -1,212 +0,0 @@
-/*
```

Comment:
We should change the main class in build.gradle files too

Reply:
It is library. Changed readme